packages/ra-ui-materialui/src/field/TranslatableFields.tsx

```diff
@@ -0,0 +1,107 @@
+import * as React from 'react';
+import { ReactElement, ReactNode } from 'react';
+import {
+    TranslatableContextProvider,
+    useTranslatable,
+    UseTranslatableOptions,
+    Record,
+} from 'ra-core';
+import { TranslatableFieldsTabs } from './TranslatableFieldsTabs';
+import { TranslatableFieldsTabContent } from './TranslatableFieldsTabContent';
+import { makeStyles } from '@material-ui/core/styles';
+import { ClassesOverride } from '../types';
+
+/**
+ * Provides a way to show multiple languages for any fields passed as children.
+ *
```

Comment:
Could you add an example value format, as in the doc? It helps understanding when to use it.

Reply:
Done